Branch-SDK/src/main/java/io/branch/referral/BranchLogger.kt

```diff
@@ -10,24 +10,32 @@ object BranchLogger {
 
     private const val TAG = "BranchSDK"
 
+    enum class BranchLogLevel(val level: Int) {
+        ERROR(1), WARN(2), INFO(3), DEBUG(4), VERBOSE(5)
+    }
+
+    @JvmStatic
+    var minimumLogLevel: BranchLogLevel = BranchLogLevel.DEBUG
```

Comment:
I would name this as `loggingLevel`, (there's no accompanying `maximum`)

Reply:
Thats fair. It's technically the minimum since all higher log levels get logged too, but I can see how the naming is confusing. I'll update that.